integration/contracts/EthLog.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity ^0.4.23;
+
+contract EthLog {
```

Comment:
Instead of making a copy of the necessary contracts, can we refer to the versions in /examples? I'm worried about keeping them in sync in the future.

Reply:
I'd have to also rewrite the examples to use the deployer script I believe, but not sure if we want to do that just now, perhaps a separate chore?